web/packages/teleport/src/Sessions/SessionList/SessionJoinBtn.tsx

```diff
@@ -84,6 +86,17 @@ export const SessionJoinBtn = ({
         showCTA={showCTA}
         closeMenu={closeMenu}
       />
+      {showModeratedCTA && (
+        <Box mx={12} my={3}>
+          <ButtonLockedFeature
+            noIcon
+            height="40px"
+            event={CtaEvent.CTA_ACTIVE_SESSIONS}
+          >
+            Join Moderated Sessions with Teleport Enterprise
```

Comment:
```suggestion
            Join as a moderator with Teleport Enterprise
```

Thoughts? It might be more obvious that this applies to the grayed-out "join as a moderator" option if we use the same language.